src/app/payment/paymentMethod/CheckoutcomCustomFormFields.tsx

```diff
@@ -28,10 +28,18 @@ interface OptionButtonProps {
     onClick?(event: SyntheticEvent<EventTarget>): void;
 }
 
-const Sepa: FunctionComponent<CheckoutcomAPMFormProps> = ({method}) => {
-    const checkoutContext = useContext(CheckoutContext);
+interface SepaCreditor {
+    sepaCreditorAddress: string;
```

Comment:
this is probably something to update at the API level as well, but prefixing every property with `sepaCreditor` seems unnecessary